mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +400,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        SharedVariables.isAlertsFragmentAvailable = false;
```

Comment:
@janakamarasena Make it instance variable as we don't care about callback when it is called from fav.